tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -1815,3 +1815,443 @@ def test_work_order_submit_requesterNonce_param_empty(self):
                 is TestStep.SUCCESS.value)
         logger.info('\t\t!!! Test completed !!!\n\n')
 
+    @pytest.mark.work_order_submit
```

Comment:
Shouldn't this be a bug? As per EEA document requesterSignature should be a  "requesterSignature": <BASE64 string>